docs/docs/querying.md

```diff
@@ -287,11 +288,18 @@ something.findOne({
     // Will order by name on an associated User
     [User, 'name', 'DESC'],
 
+    // Will order by name on an associated User using the attribute's association alias
+    ['User.name', 'DESC'],
```

Comment:
I understand.  So ['Task', 'Project', 'createdAt', 'DESC'] where the first items of the array are either a model, association, name of an association